index.html

```diff
@@ -1788,7 +1788,7 @@ <h2>
           </tr>
           <tr data-cite="orientation-event">
             <td class="string-token">
-              "[=accelerometer=]"
+              [="accelerometer"=]
```

Comment:
Seems weird to me to have the quotes be part of the definition, but I see that's what they did. 🤷